core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/HostGroupPayload.java

```diff
@@ -1,7 +1,9 @@
 package com.sequenceiq.cloudbreak.reactor.api.event;
 
+import java.util.Set;
+
 import com.sequenceiq.cloudbreak.common.event.Payload;
 
 public interface HostGroupPayload extends Payload {
-    String getHostGroupName();
+    Set<String> getHostGroupNames();
```

Comment:
Maybe better to leave this as is, and introduce a HostGroupsPayload / HostMultipleGroupPayload?
Guessing there's some operations which continue to operate on a single hostGroup only, and would be easier to understand this if the implemented interface was different.